### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by Alexr16**
In this PR:
- Removed all hardcoded items from the tasks array.
- Implemented a function for adding a new task to the array.
- Implemented a function for deleting a task from the array.
- Implemented a function for editing task descriptions.
- Added function to update items indexes when a task is deleted. 
- Data is preserved in the local storage.

### CODE REVIEW

File: src/modules/list.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    //       <button class="menu" id="menu-btn"></button>
+    // `;
+    // ul.appendChild(li);
+    // ul.insertBefore(li, ul.childNodes[0]);
+    // document.querySelector('.item-text').addEventListener('click', this.editEvent());
+    // // document.querySelectorAll('.item-text').forEach((n) => n.addEventListener('click', () => {
+    // //   input.removeAttribute('readonly');
+    // //   const menu = document.querySelectorAll('.menu');
+    // //   const edit = document.querySelectorAll('.to-do');
+    // //   menu.classList.toggle('menu');
```
Comment (micheaol) [inline src/modules/list.js]: Please ensure that all commented codes are removed from your codebase. It makes your codebase clean.

File: src/modules/store.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    list.push(task);
+    localStorage.setItem('task', JSON.stringify(list));
+  }
+
+  static remove(index) {
+    const list = Store.getList();
+    list.forEach((task) => {
+      if (task.index === index) {
+        list.splice(index - 1, 1);
+        // list = list.filter((task) => task.index !== index);
```
Comment (micheaol) [inline src/modules/store.js]: Please ensure that all commented codes are removed from your codebase. It makes your codebase clean.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Alexr16 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Alexr16** [comment]
- Removed all commented codes.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Alexr16, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**